src/App.tsx

```diff
@@ -17,18 +66,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                sortBy={sortBy}
+                setSortBy={setSortBy}
```

Comment:
Check point 3 in checklist, you should use correct naming for function with key word 'on'